### PR TITLE
Executor Plugin: Improvements and fixes

### PR DESCRIPTION
**Pull request by onkelandy**
Major fix for autocomplete
Performance improvements (loading autocomplete takes 15s for me and blocked the webif..)
Sorting option for saved files

### CODE REVIEW

**Comment by bmxp** [comment]
It looks fine, just go ahead and merge it. Thank you for the improvements!

**Comment by bmxp** [comment]
Maybe we can find a convenient way to deliver some sample skripts as well. Do you have an idea? Maybe a subdir within the plugin folder which will be merged into the list and that are readonly and colorized within the list?

**Comment by onkelandy** [comment]
Including "standard" examples in a sub-dir sounds fine for me. Also the idea of putting them in the list with different color (grey?, maybe italic font?). Just the 3 examples from the documentation?

File: executor/user_doc.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ executor
 Einführung
 ~~~~~~~~~~
 
-Das executor plugin kann genutzt werden, um **Python Code** (z.B. für **Logiken**) und **eval Ausdrücke** zu testen.
+Das executor Plugin kann genutzt werden, um **Python Code** (z.B. für **Logiken**) und **eval Ausdrücke** zu testen.
```
Comment (bmxp) [inline executor/user_doc.rst:19]: Das mit den eval Ausdrücken kann raus, die sind ja im Admin Interface bereits implementiert.

File: executor/webif/__init__.py [hunk 2]
```diff
@@ -197,7 +197,7 @@ def eval_statement(self, eline, path, reload=None):
     def exec_code(self, eline, reload=None):
         """
         evaluate a whole python block in eline
-                              
+
         :return: result of the evaluation
         """
         result = ""
```
Comment (bmxp) [inline executor/webif/__init__.py:203]: der stub_logger müsste noch um neuere methoden wie notice etc. erweitert werden...

**Review by bmxp — APPROVED** [review]

**Pull request merged**